test/support/event_handler.ex

```diff
@@ -32,6 +32,11 @@ defmodule Test.Event.Handler do
     {:merge, :all, :app_state, data}
   end
 
+  def handle_event(%Brook.Event{type: "UNKNOWN"} = event) do
+    IO.inspect(event, label: "Brook Event")
```

Comment:
Does Credo complain about this? Will he when we pull it in as a dependency to a project that runs mix credo?